src/components/inputs/TextInput.js

```diff
@@ -1,9 +1,9 @@
 import { useContext } from "react";
-import { inputContext } from "../../App";
+import { InputContext } from "../../App";
 import "./inputStyles.css";
 
 const TextInput = ({ type, placeholder, label, id, textarea }) => {
```

Comment:
Can we use something like:

```
const TextInput = ({...rest}) => {
   return <input {...rest />
```
So we are not limited to only those properties.

Reply:
I'd leave it as for now until we have a use case for more props. Changes here would be outside the scope specified in the tickets, so if we do want to make changes it should be in a new ticket.